src/cdk-experimental/menu/menu-item.ts

```diff
@@ -35,15 +36,33 @@ export class CdkMenuItem {
   }
   private _disabled = false;
 
+  /**
+   * If this MenuItem is a regular MenuItem, outputs when it is triggered by a keyboard or mouse
+   * event.
+   */
+  @Output('cdkMenuItemOnTrigger') _trigger: EventEmitter<void> = new EventEmitter();
```

Comment:
Outputs are typically considered public API, so the property wouldn't have an underscore prefix